pywbemtools/pywbemcli/_common.py

```diff
@@ -456,6 +456,51 @@ def create_cimproperty(cim_type, is_array, name, value_str):
     return CIMProperty(name, cim_value, cim_type)
 
 
+def create_ciminstancename(cim_class, kv_keys):
+    """
+    Create a cim instance from the input parameters.
+
+      Parameters:
+
+        cim_class_: (:Class:`pywbem.CIMClass`)
+            The class from which the CIMInstance is to be created
+
+        properties ():
+            A tuple of name/value pairs representing the properties and their
+            values that are to be constructed for the instance. Required
+
+      Returns: CIMInstanceName with namespace = None and host = None
+
+      Exceptions:
+        click.ClickException if Property name not found in class or if mismatch
+          of property type in class vs value component of kv pair
+    """
+    # TODO make usable if no class can be acquired because GetClass not
+    # implemented in server.
+    keys = NocaseDict()
+    for kv_property in kv_keys:
+        name, value_str = parse_kv_pair(kv_property)
+        try:
+            cl_prop = cim_class.properties[name]
+        except KeyError:
+            raise click.ClickException('Property name "{}" not in class "{}".'
+                                       .format(name, cim_class.classname))
+
+        try:
+            if value_str.startswith('"') and value_str.endswith('"'):
+                value_str = value_str[1:-1]
```

Comment:
How about `value_str = value_str.strip('"')`?

Reply:
1. That would actually have to be more complex because it would have to account for escaped quotes inside the string.  This simply removes the lead and end if they both exist which is what we want to do because otherwise the create_cimvalue gets confused and adds more quotes.

2. No problem moving outside try block.  **DONE**

Reply:
`strip()` does remove only from start and end.